compiler/parser/src/conversions.rs

```diff
@@ -679,12 +688,91 @@ pub fn to_expression(node: &SyntaxNode<'_>, builder: &NodeBuilder, handler: &Han
                     id: builder.next_id(),
                 }
                 .into()
+            } else if let (1, Some(leo_ast::CoreFunction::Get)) =
+                (args.len(), leo_ast::CoreFunction::from_symbols(Symbol::intern("unresolved"), name.name))
+            {
+                // We use `unresolved` here because a `Get` might refer to `Vector::get` or
+                // `Mapping::get`. No way to know just yet.
+                leo_ast::AssociatedFunctionExpression {
+                    variant: leo_ast::Identifier::new(Symbol::intern("unresolved"), builder.next_id()),
```

Comment:
Does `unresolved` need to be prefixed with something so that it wouldn't conflict with a user declared variable?

Reply:
Hmm valid concern. I'll do that just in case.

Reply:
Done